src/Dashboard/Grid.php

```diff
@@ -423,7 +423,7 @@ public function show(bool $mini = false)
 
         $js = <<<JAVASCRIPT
       $(function () {
-         Dashboard.display({
+        Dashboard.dashboards[$rand] = new GLPIDashboard({
```

Comment:
Maybe it could be possible to do this in dashboard constructor, to prevent having to set it manually everytime `new GLPIDashboard()` is used.
```
Dashboard.dashboards[options.rand] = this;
```